### PR TITLE
[quant] Enable backward for choose_qparams_per_token_asymmetric

### PR DESCRIPTION
**Pull request by andrewor14**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #123452

Summary: When running the backward for this op, we get the error:
```
RuntimeError: derivative for aten::aminmax is not implemented
```
This commit replaces this call with separate amin and amax
calls instead, which do have implemented derivatives.

Test Plan:
python test/test_quantization.py -k test_decomposed_choose_qparams_per_token_asymmetric_backward

Reviewers: jerryzh168, digantdesai

Subscribers: jerryzh168, digantdesai, supriyar

Differential Revision: [D55805170](https://our.internmc.facebook.com/intern/diff/D55805170)

### CODE REVIEW

**Comment by andrewor14** [comment]
@andrewor14 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D55805170).

**Comment by andrewor14** [comment]
@andrewor14 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D55805170).

**Review by digantdesai — APPROVED** [review]

File: test/quantization/core/test_quantized_tensor.py [hunk 1]
```diff
@@ -1602,6 +1602,14 @@ def test_decomposed_dequantize_per_channel(self):
         self.assertEqual(quantized_X.int_repr(), quantized_decomposed_X)
         self.assertEqual(dequantized_X, dequantized_decomposed_X)
 
+    def test_decomposed_choose_qparams_per_token_asymmetric_backward(self):
```
Comment (jerryzh168) [inline test/quantization/core/test_quantized_tensor.py:1605]: is this used in QAT path? like choose_qparams -> fake_quant_op?

Comment (andrewor14) [reply test/quantization/core/test_quantized_tensor.py:1605]: yeah

**Review by jerryzh168 — APPROVED** [review]

**Review by zou3519 — APPROVED** [review]

**Pull request closed without merging**